.github/ISSUE_TEMPLATE/react-native-windows--vnext--module-request.md

```diff
@@ -17,6 +17,14 @@ assignees: ''
 - Target version: 
 - [ ] Issue exists on module repo (if so, [link]())
 <!-- If the community module repo already has an issue tracking adding React Native Support, please check this box and link to it. -->
+- [ ] Module has a vCurrent/C# implementation (if so, [link]())
```

Comment:
vCurrent is essentially an internal codename. I don't think it'd necessarily be understood by people outside of our internals. Do we have a better term for this?

Reply:
We've started calling vcurrent "legacy"

Reply:
I'll swap from vCurrent to "Legacy".